rollup.config.js

```diff
@@ -17,7 +17,7 @@ if you want to view the source visit the plugins github repository
 export default {
 	input: 'src/main.ts',
 	output: {
-		dir: './dist',
+		dir: process.env.BUILD === 'development' ? '.' : './dist',

```

Comment:
What's the reason for that?
I routinely copy `dist` to the Obsidian plugins plugin directory to test as I'm developing, why ever build to `.`?

Reply:
I have the whole repository in the obsidian plugin directory.
When changes are made, the compiler automatically builds them and restarting obsidian shows the changes.
No need to copy the files out of the dist directory. They just work in place.

Reply:
This workflow is described in the sample plugin and is how the example works.
https://github.com/obsidianmd/obsidian-sample-plugin#first-time-developing-plugins

Reply:
I see; it is meant to live inside the `.obsidian/plugins` directory itself.
I rather not work this way; how about making an additional command that is meant to do this (e.g. `dev-inline`), *or* make this behavior the dev default, but make a `dev-dist` command that I can use to keep my existing flow?

Reply:
What is different between your workflow and the build command?